sdks/java/core/src/test/java/org/apache/beam/sdk/coders/CoderRegistryTest.java

```diff
@@ -89,8 +90,14 @@ public void testSerializableFallbackCoderProvider() throws Exception {
   public void testProtoCoderFallbackCoderProvider() throws Exception {
     CoderRegistry registry = getStandardRegistry();
 
+    //must use reflection here or a reference to dataflow class is baked into
```

Comment:
With tycho 0.26.0 on my Mac, if I revert the CoderRegistryTest, I get the same error as with Tycho 0.24.0.   

In ALL cases, I don't see any additional tests running.  Tycho 0.24, Tycho 0.26, and "javac" all result in 347 tests run:

```
Tests run: 347, Failures: 0, Errors: 0, Skipped: 3
```

With either tycho and the reverted test, I see:

```
Tests in error:
  CoderRegistryTest.testIgnoredGenericInPipeline:449 » NoClassDefFound com/googl...
  CoderRegistryTest.testSpecializedButIgnoredGenericInPipeline:420 » NoClassDefFound

Tests run: 347, Failures: 0, Errors: 2, Skipped: 3
```

I'm really confused as to why you think those two tests in CoderRegiseterTest shouldn't be run.   In my case, they are always run as part of the direct-java integration test.  If I don't use a profile, they run.  If I use the profile, they run.   They just happen to pass with javac compiled test class, but not with the eclipse compiled test class.  Note: it's the compiler that compiles sdk/java/core that matters, not the one that compiles runners/direct-java.